datalad/support/tests/test_annexrepo.py

```diff
@@ -1837,7 +1835,7 @@ def _test_status(ar):
 
     def sync_wrapper(push=False, pull=False, commit=False):
         # wraps common annex-sync call, since it currently fails under
-        # mysterious circumstances in V6 adjusted branch setups
+        # mysterious circumstances in V6+ adjusted branch setups
```

Comment:
just for consistency
```suggestion
        # mysterious circumstances in v6+ adjusted branch setups
```

Reply:
v6 versus V6 is already used inconsistently across the codebase.  Switching one instance doesn't improve the situation.